### DISCUSSION TITLE
The label isn't displayed in a self-generated circuit.

### DISCUSSION THREAD

**Discussion by effha** [hneemann/Digital#1135] [Q&A]
An SVG template is created with a label. Is it possible to display this? I used Incscape to move the label into the circuit frame. But the part in the library contains only the empty frame without the label.

**Comment by hneemann** [comment]
If a text is present in an svg file, this text should also be part of the shape. But maybe the svg importer does not find the text if the structure of the svg file is too complex. The svg importer is quite decent but far from perfect.

It is also possible to change the ID of the pin circle in the svg file: If the existing `id="pin:[name]"` is expanded to `id="pin+:[name]"` the label will be set exactly like the normal labels of the shapes.
